### ISSUE TITLE
injection order

### ISSUE THREAD

**Issue by Hany202020** [ElucidataInc/ElMaven#1352]
on V12 for mac, the injection order button doesnot work !

**Comment by saifulbkhan** [comment]
@Hany202020 Yes, the sorting does not work, right. Really sorry about that. We plan to release another patched version (v0.12.1) very soon, where this should be fixed.
